content/hero.yaml

```diff
@@ -1,13 +1,18 @@
 headline: A free, cross-platform desktop client for the Graasp ecosystem.
 
-body: Aenean condimentum, lacus sit amet luctus lobortis, dolores et quas molestias excepturi enim tellus ultrices elit, amet consequat enim elit noneas sit amet luctu. Quis nostrum exercitationem ullam corporis suscipit laboriosam.
+body: Download open source Graasp Desktop for you preferred operating system.
```

Comment:
Download the open source, community version of Graasp Desktop for you preferred operating system.